oak-security-spi/src/main/java/org/apache/jackrabbit/oak/spi/security/authentication/AuthInfoImpl.java

```diff
@@ -64,7 +64,7 @@ public static AuthInfo createFromSubject(@NotNull Subject subject) {
 
     @Override
     public String toString() {
-        return Objects.toStringHelper(this)
+        return MoreObjects.toStringHelper(this)
```

Comment:
why is it MoreObjects and not Objects?

Reply:
:-) Once again, the Guava developers moved that method. Believe me.